java/src/com/swiftnav/sbp/drivers/SBPDriverJSSC.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2015 Swift Navigation Inc.
+ * Contact: Gareth McMullin <gareth@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+package com.swiftnav.sbp.drivers;
+
+import com.swiftnav.sbp.client.SBPDriver;
+
+import java.io.IOException;
+
+import jssc.SerialPort;
+import jssc.SerialPortException;
+
```

Comment:
Was this stuff taken from the old JVM PR? If so, we probably ought to credit the forum post by this guy: https://github.com/swift-nav/libsbp/issues/147, where some of the original PR was taken. Unfortunately, the download link from that post doesn't seem to work anymore.